### PR TITLE
annotated diffusion: removed convnext and synched with notebook

### PR DESCRIPTION
**Pull request by kashif**
notebook PR is at: https://github.com/huggingface/notebooks/pull/288

### CODE REVIEW

File: annotated-diffusion.md [hunk 1]
```diff
@@ -225,21 +245,45 @@ class SinusoidalPositionEmbeddings(nn.Module):
         return embeddings
 ```
 
-### ResNet/ConvNeXT block
+### ResNet block
 
-Next, we define the core building block of the U-Net model. The DDPM authors employed a Wide ResNet block ([Zagoruyko et al., 2016](https://arxiv.org/abs/1605.07146)), but Phil Wang decided to also add support for a ConvNeXT block ([Liu et al., 2022](https://arxiv.org/abs/2201.03545)), as the latter has achieved great success in the image domain. One can choose one or another in the final U-Net architecture.
+Next, we define the core building block of the U-Net model. The DDPM authors employed a Wide ResNet block ([Zagoruyko et al., 2016](https://arxiv.org/abs/1605.07146)), but Phil Wang has replaced the standard convolution in the projection layer to `WeightStandardizedConv2d` which works better with Group norm:
```
Comment (NielsRogge) [inline annotated-diffusion.md:250]:
```suggestion
Next, we define the core building block of the U-Net model. The DDPM authors employed a Wide ResNet block ([Zagoruyko et al., 2016](https://arxiv.org/abs/1605.07146)), but Phil Wang has replaced the standard convolutional layer by a "weight standardized" version, which works better in combination with group normalization (see [Kolesnikov et al., 2019](https://arxiv.org/abs/1912.11370) for details).
```

**Review by NielsRogge — APPROVED** [review]
Thanks for improving!

File: annotated-diffusion.md [hunk 2]
```diff
@@ -247,7 +247,7 @@ class SinusoidalPositionEmbeddings(nn.Module):
 
 ### ResNet block
 
-Next, we define the core building block of the U-Net model. The DDPM authors employed a Wide ResNet block ([Zagoruyko et al., 2016](https://arxiv.org/abs/1605.07146)), but Phil Wang has replaced the standard convolution in the projection layer to `WeightStandardizedConv2d` which works better with Group norm:
+Next, we define the core building block of the U-Net model. The DDPM authors employed a Wide ResNet block ([Zagoruyko et al., 2016](https://arxiv.org/abs/1605.07146)), but Phil Wang has replaced the standard convolutional layer by a "weight standardized" version, which works better in combination with group normalization (see [Kolesnikov et al., 2019](https://arxiv.org/abs/1912.11370) for details).
```
Comment (NielsRogge) [inline annotated-diffusion.md:250]:
```suggestion
Next, we define the core building block of the U-Net model. The DDPM authors employed a Wide ResNet block ([Zagoruyko et al., 2016](https://arxiv.org/abs/1605.07146)), but Phil Wang has replaced the standard convolutional layer by a "weight standardized" version, which works better in combination with group normalization (see ([Kolesnikov et al., 2019](https://arxiv.org/abs/1912.11370)) for details).
```
Sorry made a typo here :)
